membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -286,24 +299,82 @@ def test_analysis_get_z_surface(self, universe, x_bin, y_bin, expected_surface):
         avg_surface = mc.results.average_z_surface
         assert_almost_equal(avg_surface, expected_surface)
 
-    def test_analysis_mean(self, universe):
```

Comment:
Instead of deleting this test and replacing it with the  new one, could you please keep both tests for `wrap=True` and `wrap=False`?

Reply:
Ok, I added a test for both. Thanks for this. :)